src/neoxp/TransactionExecutor.cs

```diff
@@ -278,7 +278,14 @@ static async Task WriteStackItemAsync(System.IO.TextWriter writer, Neo.VM.Types.
                         await WriteLineAsync(Neo.Helper.ToHexString(buffer.GetSpan())).ConfigureAwait(false);
                         break;
                     case Neo.VM.Types.ByteString byteString:
-                        await WriteLineAsync(Neo.Helper.ToHexString(byteString.GetSpan())).ConfigureAwait(false);
+                        if (byteString.GetSpan().TryGetUtf8String(out var text))
+                        {
+                            await WriteLineAsync($"{Neo.Helper.ToHexString(byteString.GetSpan())}({text})").ConfigureAwait(false);
```

Comment:
I think text 1st then hex. Or put on different lines when printing.

Example: `Return: "Hello World" (Hex-Format: "48656c6c6f20576f726c64")'

Reply:
It is not a accurate convert, some random data could be converted to string. So I prefer show its text as a possible value after the certain value(HexString).

Reply:
Yes but you are showing string and hex-string. Think it neater for it to look like this `"Hello World" (Hex-Format: "48656c6c6f20576f726c64")` and more understandable.

Reply:
You have the `StackItem`, why not use `StackItem.GetString()`? that will fix this problem.

Reply:
`StackItem.GetString()` will throw exception if it is not a valid string.